reader-graphml/src/main/java/com/calpano/graphinout/reader/graphml/GraphmlSAXHandler.java

```diff
@@ -74,7 +60,10 @@ public void characters(char[] ch, int start, int length) throws SAXException {
                 throw new IllegalStateException("No open element to add characters to.");
             }
         } else {
-            openEntities.peek().addCharacters(new String(ch, start, length));
```

Comment:
This is not correct in general. Different elements must handle whitespace differently. E.g. when in <data>, we must keep all data 1:1 as it was. Only inside elements where "no characters" is expected we can omit the whitespace. Character handling should happen in the GioXEntity classes.